### PR TITLE
Option to disbale events by tags

### PR DESCRIPTION
**Pull request by chbatey**
Refs #271

### CODE REVIEW

File: core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -156,6 +156,10 @@ cassandra-journal {
   cassandra-2x-compat = off
 
   events-by-tag {
+    # Enable/disable events by tag. If eventsByTag queries aren't required then this should be set to
+    # false to avoid the overhead of maintaining the tag_views table.
+    enabled = true
```
Comment (johanandren) [inline core/src/main/resources/reference.conf]: Shouldn't it be false by default then as that is likely what you'd start out with (and then later turn it on when starting to use tags)?

Comment (chbatey) [reply core/src/main/resources/reference.conf]: I am on the fence here, as building up the tag_views can be very expensive (and only possible via the migration tools) if users have a lot of events. Where as if they notice the excess writes to the table then they can turn it off and truncate it.

Comment (johanandren) [reply core/src/main/resources/reference.conf]: Ok, sounds reasonable to have it enabled by default then.

Comment (patriknw) [reply core/src/main/resources/reference.conf]: I agree

**Pull request merged**